README.md

```diff
@@ -1,5 +1,130 @@
-# Hackathon
+# Global Legal Hackathon
 
 <br>
 
+### Project Contacts
+
+Matthew Schneider
+    [INSERT]
+
+Jordan L. Fischer, Esq.
+    Teaching Professor of Law
+    Thomas R. Kline School of Law at Drexel University
+    [jlf324@drexel.edu](jlf324@drexel.edu)
+
+Cameron Bale
+    [INSERT]
+
+### Location-Based Data Privacy Solution
+
+This Project addresses the trade-off between the accuracy of location data with privacy and legal privacy protections.
+
+Prior research (de Montjoye et al. 2013) shows that four spatio-temporal points are enough to uniquely identify 95% of individuals. This means that the assortment of COVID-19 location-tracking apps are destined to identify some individuals. Privacy concerns are further exacerbated when geospatial points can be traced to home addresses which are linked to public tax and real estate sale records.
+
+### Project Goals
+
+For the purpose of this FT Global Legal Hackathon, this project addresses the following core components: 
+
+* Statistical process to reduce the identifiability of individuals while maintaining most of the usefulness of the data
+
+* Differentially private histogram approach for geospatial data which provides a theoretical guarantee to individual privacy.
+
+### Proposed Solution
+
+
+#### Project Assumptions & Out of Scope Matters
+
+Due to the shortened timeline of this project, our team made a number of assumptions and selected to only address a sub-set of the issues related to the trade-off between data location and privacy.
+
+1. Varying Legally Defined Terms 
+
+The key terms used in this project (see below) have a vareity of different defintions. Our project created standard defintions from those sources, but we recognize that these terms are arguablly nuanced. As such, in our Future Work, we will look to further explore these various defintions.
+
+2. INSERT
+
+### Legal Framework with Related Statistical Analysis 
+
+The use of location data as solution to address the spread of COVID-19 directly impacts interrelated but distinctly different terms under privacy law: 
+
+* Pseudonymization;
+* De-identification;
+* Anonymization; and
+* Aggregation.
+
+These terms are similar and often used interchangeably. The terms generally address the core idea of removing identifiers from data. However, with regulations and standards, each termcan be defined and used distinctinly differentlly. 
+
```

Comment:
Should we also say that each definition of the same term can restrict how location apps are used depending on the applicable law?   
This leads to the reason why we present 4  different use metrics in this paper .

Reply:
Agreed. Added.